sdcm/report_templates/results_latency_during_ops_short.html

```diff
@@ -78,21 +78,24 @@ <h2>{{ operation }}</h2>
                                 <th>Latency {{ lat_type }}</th>
                                 <th>Steady State {{lat_type}}</th>
                             {% endfor %}
+                            <th> Duration (sec) </th>
 
                             <th>Commit id, date</th>
                         </tr>
                         {% for cycle in results["cycles"] %}
                             {% set row_span = cycle["hdr_summary"]|length %}
+                            {% set cycle_index = loop.index %}
                             {% for workload in cycle["hdr_summary"] %}
                                 <tr>
                                     {% if loop.first %}
-                                            <td rowspan="{{ row_span }}"> Cycle #{{ loop.index }} </td>
+                                            <td rowspan="{{ row_span }}"> Cycle #{{ cycle_index }} </td>
                                             <td rowspan="{{ row_span }}"> current build </td>
                                             <td> {{ workload }}</td>
                                             {% for perc in lat_type_list %}
-                                                <td style="background-color: {{ cycle['hdr_summary'][workload]['color'][perc] }}"" }};"> {{ cycle['hdr_summary'][workload][perc]}} </td>
+                                                <td style="background-color: {{ cycle['hdr_summary'][workload]['color'][perc] }};"> {{ cycle['hdr_summary'][workload][perc]}} </td>
```

Comment:
what is this change about? the table's header?

Reply:
this is fix of invalid syntax